### DISCUSSION TITLE
正文字号

### DISCUSSION THREAD

**Discussion by obstacle213** [sjtug/SJTUThesis#863] [提问]
请问如果想要修改正文字号的话应该在哪里加什么语句？
谢谢~

**Comment by LogCreative** [comment]
文档类选项上设定 `zihao` 参数。

https://github.com/sjtug/SJTUThesis/blob/700093d999dc6fe212aa280d04aee6c120470f3b/main.tex#L7

例如：
```latex
\documentclass[zihao=-4]{sjtuthesis}
```
